go/consensus/tendermint/abci/prune.go

```diff
@@ -60,111 +60,91 @@ type PruneConfig struct {
 	Strategy PruneStrategy
 
 	// NumKept is the number of versions retained when applicable.
-	NumKept int64
+	NumKept uint64
 }
 
-// StatePruner is a concrete ABCI mux iAVL state pruner implementation.
+// StatePruner is a concrete ABCI mux state pruner implementation.
 type StatePruner interface {
-	// Prune purges unneeded versions from the ABCI mux iAVL tree,
+	// Prune purges unneeded versions from the ABCI mux node database,
 	// given the latest version, based on the underlying strategy.
-	Prune(latestVersion int64)
-
-	// Subscribe subscribes to prune events.
-	Subscribe() (<-chan int64, *pubsub.Subscription, error)
+	Prune(ctx context.Context, latestRound uint64) error
 }
 
 type statePrunerInitializer interface {
-	Initialize(latestVersion int64) error
+	Initialize(latestRound uint64) error
```

Comment:
Because our storage interface has a concept of rounds instead of versions (as it was previously only used for our runtime state). It is the same thing though.

Reply:
Renamed it back.